src/components/llama-item.ts

```diff
@@ -56,6 +56,27 @@ export class LlamaItem extends LitElement {
     }
   }
 
+  /**
+   * Deletes an item from storage and removes it from the UI.
+   *
+   * @todo Should the db interaction be happening here or in llaminator.ts? Same with
+   *     shareItem() - should it be an event that llaminator.ts handles?
```

Comment:
High-level thoughts on this: we aren't currently following any particular architecture paradigms, but I think that moving towards MVVM would align well with what we need.